src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -15,45 +15,81 @@ import {
   FacetRefinement,
   NumericRefinement,
 } from '../../lib/utils/getRefinements';
-import { Connector, TransformItems } from '../../types';
+import { Connector, TransformItems, CreateURL } from '../../types';
 
-export type ConnectorRefinement = {
+export type CurrentRefinementsConnectorParamsRefinementType =
+  | 'facet'
+  | 'exclude'
+  | 'disjunctive'
+  | 'hierarchical'
+  | 'numeric'
+  | 'query'
+  | 'tag';
+
+export type CurrentRefinementsConnectorParamsRefinement = {
```

Comment:
Previously we also exported a normalised type, I decided to remove it to make things simple.

Reply:
Will users have to cast the type as [we're doing in the tests](https://github.com/algolia/instantsearch.js/blob/453eed3d98f3ba3d8ffa345ad3b50bffb4cb0d1a/src/components/CurrentRefinements/__tests__/CurrentRefinements-test.tsx#L324)?

Reply:
Well, yes.

At least on the experimental yes, because we are not exporting every single type yet.

Reply:
Moving it to `string` meanwhile.

Reply:
It seems that the users have to do some casting at some point or they will get in that case: https://stackoverflow.com/questions/37978528/typescript-type-string-is-not-assignable-to-type since 

If we do not want to weaken the type, the simplest way for them to cast it is to use the `const` assertion ( https://devblogs.microsoft.com/typescript/announcing-typescript-3-4-rc/ ), this way they do not need to type to be exposed.